### PR TITLE
Filter route watch to only "console"

### PR DESCRIPTION
**Pull request by benjaminapetersen**
There is no reason to watch any other route.

### CODE REVIEW

**Comment by benjaminapetersen** [comment]
@enj fyi

**Comment by benjaminapetersen** [comment]
/retest

throttled.

File: pkg/console/starter/starter.go [hunk 1]
```diff
@@ -83,6 +83,10 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 		options.FieldSelector = fields.OneTermEqualSelector("metadata.name", api.OAuthClientName).String()
 	}
 
+	tweakListOptionsForRoute := func(options *v1.ListOptions) {
+		options.FieldSelector = fields.OneTermEqualSelector("metadata.name", api.OpenShiftConsoleRouteName).String()
```
Comment (enj) [inline pkg/console/starter/starter.go]: Please stop making different constants for the same string.  You do not even use them consistently:

`operator.WithInformer(routes, operator.FilterByNames(api.OpenShiftConsoleName)),`

Comment (benjaminapetersen) [reply pkg/console/starter/starter.go]: The `const` for `OpenShiftConsoleRouteName ` is not new.  It's old, from when we didn't have all matching names.  I'd rather use this one to match what is set & do a cleanup later & eliminate all the extra `const`s.  I believe we are down to a single name `console` for everything except the `configmaps` at this point.

**Pull request merged**